### PR TITLE
docs: add pr handling section to contribution.md

### PR DESCRIPTION
**Pull request by karlbaumhauer**
## Description:
Section about handling PRs in our Project has been added.

Closes https://github.com/solid-design-system/internal/issues/45

## Definition of Reviewable:
- [x] Documentation is created/updated
- [x] relevant tickets are linked
- [x] PR is assigned to project board

### CODE REVIEW

**Comment by karlbaumhauer** [comment]
unfortunately, it does not work like we want it to do...

The assignee section is restricted to a max of 10 persons and seem to not provide the possibility to assign teams. This means every author needs to know the team members of the respective teams.

The reviewers section would allow a max of 15 and has the option to request a team to review.

**Comment by Vahid1919** [comment]
> unfortunately, it does not work like we want it to do...
> 
> The assignee section is restricted to a max of 10 persons and seem to not provide the possibility to assign teams. This means every author needs to know the team members of the respective teams.
> 
> The reviewers section would allow a max of 15 and has the option to request a team to review.

@karlbaumhauer When I first started working at SDS, I used the open PRs to see who I should assign/request review from. The points you mentioned are definitely valid though. 

From what I understand, temp developers also can review, so it should just be a matter to communicate/document that we should assign the respective developers/designers we are requesting a review from. And that they could take a look at open PRs to see who they should assign.

**Comment by karlbaumhauer** [comment]
Your right @Vahid1919. This would solve the issue on who should be assigned. 

We still have the problem that authors need to assign every single person manually which seems to be a lot of clicking and checking another or who is in which team etc.

**Comment by mariohamann** [comment]
Don't see an issue here. Sometimes people are on vacation etc. so you even don't need everone.

our team is not that big that I see a big hustle there. At least I never did.

**Review by coraliefeil — APPROVED** [review]
All good THX

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +64,27 @@ Each new component in the Solid Design System, representing an old component fro
...
+3. Reviewers will unassign themselves after they have reviewed the PR
+4. If changes are requested, the author needs to reassign the reviewer after the changes are made and pushed
+
+> _**NOTE:**_
+> If UI changes need to be reviewed in Chromatic, the reviewer need to add a comment to the PR, stating if the changes are approved or new changes are requested. This is mandatory as the Chromatic approval will not send any notifications.
+
+Assigning reviewers follows a few rules:
+
+1. **New Feature PRs:** All current members from the SDS team need to be assigned to the PR (@solid-design-system/current-sds-team).
+2. **Bug Fix or Docs PRs:** Minimum 1 developer from the SDS dev-core-team (@solid-design-system/core-development) needs to be added as reviewer. Designers need to be added separately if visual changes are made.
```
Comment (karlbaumhauer) [inline CONTRIBUTING.md:85]:
```suggestion
2. **Bug Fix or Docs PRs:** Minimum 1 developer from the SDS dev-core-team (@solid-design-system/core-development) needs to be assigned to the PR. Designers need to be added separately if visual changes are made.
```

**Review by yoezlem — APPROVED** [review]

**Review by EllaELSA — APPROVED** [review]
Fine for me

**Review by van-nguyen-ht — APPROVED** [review]

**Review by azraefendic — APPROVED** [review]

**Review by DanielHargesheimer — APPROVED** [review]

**Review by abudd1094 — APPROVED** [review]

**Comment by mariohamann** [comment]
LGTM.

**Pull request merged**